packages/app/src/vis-packs/core/line/MappedLineVis.tsx

```diff
@@ -28,7 +29,7 @@ interface Props {
   valueLabel?: string;
   valueScaleType?: ScaleType;
   errors?: NumArray;
-  auxiliaries?: NumArray[];
+  auxiliaries?: Auxiliary[];
```

Comment:
This and #1029 makes me wonder if it would not be time to review the API of `LineVis` to have auxiliary signals first-class data arrays with the same treatment.

But I think it will be easier once #1029 is dealt with. So for now, let's keep it even though it is not ideal. Perhaps rename one `AuxParams` as it is close to `AxisParams` ?